### PR TITLE
Don't need to open wfGetDB twice

### PR DESCRIPTION
**Pull request by paladox**
Reduces duplications.

### CODE REVIEW

**Comment by JohnFLewis** [comment]
Setting $this->cwdb at the first call would be more efficient

File: includes/WikiManager.php [hunk 1]
```diff
@@ -5,17 +5,16 @@
 
 class WikiManager {
 	private $dbname = null;
-	private $dbw = null;
```
Comment (JohnFLewis) [inline includes/WikiManager.php:8]: Why?

Comment (paladox) [reply includes/WikiManager.php:8]: It's not used from what i can tell.

Comment (JohnFLewis) [reply includes/WikiManager.php:8]: Yes it is?

Comment (paladox) [reply includes/WikiManager.php:8]: Where? Searching the file i see no more occurrences of $dbw?

Comment (JohnFLewis) [reply includes/WikiManager.php:8]: Line 55

Comment (paladox) [reply includes/WikiManager.php:8]: Oh damn. Ok putting it back.

**Pull request merged**